crawl-ref/source/stairs.cc

```diff
@@ -399,6 +400,22 @@ static void _rune_effect(dungeon_feature_type ftype)
     }
 }
 
+static void _gauntlet_effect()
+{
+    // already doomed
+    if (you.species == SP_FORMICID)
+        return;
+
+    mprf(MSGCH_WARN, "The nature of this place prevents teleportation.");
+
+    if (you.has_mutation(MUT_TELEPORT, true)
+        || you.wearing(EQ_RINGS, RING_TELEPORTATION, true)
+        || you.scan_artefacts(ARTP_CAUSE_TELEPORTATION, true))
+    {
+        mpr("Your feel stable.");
+    }
+}
```

Comment:
Hmm I wonder: do you think this "You feel stable" message is necessary? I think it might just confuse people with -Tele items/teleportitis—they won't necessarily know why this message is being printed. Just putting it out there.

Reply:
I believe they will know why the message is being printed because it comes immediately after the warning. Originally I used `You feel temporarily stable` but I thought it could be misinterpreted as something related to time/delay.

Reply:
I have changed the message to reflect that it only affects the current floor.